telegram/ext/jobqueue.py

```diff
@@ -21,10 +21,22 @@
 import logging
 import time
 import warnings
+import datetime
+from enum import Enum
 from threading import Thread, Lock, Event
 from queue import PriorityQueue, Empty
 
 
+class Days(Enum):
```

Comment:
Weeks unfortunately don't always start on a monday. In some locales it's sundays.